jenkins-dsl/cassandra_job_dsl_seed.groovy

```diff
@@ -573,10 +573,11 @@ cassandraBranches.each {
                             }
                         }
                         postBuildTask {
-                            // the pgrep needs to catch any other build/process that is using docker
+                            // docker needs to (soon or later) prune its volumes too, but that can only be done when the agent is idle
```

Comment:
so when the agent is idle we remove docker volumes...
... but when it is not idle then ?
what actually we do with `docker system prune --all --force --filter `until=...` ?

Maybe worth explaining the rationale in the comment in L576?

Reply:
volumes can't be removed with the until filter is used. 

so if the agents is otherwise busy, we only removed stuff that's more than 18 hours old (no build is running that long).

when the agent is idle, we know we can wipe it all.

Reply:
comment added/extended.